Doc/whatsnew/3.14.rst

```diff
@@ -612,6 +612,12 @@ Deprecated
   write new code.  The :mod:`subprocess` module is recommended instead.
   (Contributed by Victor Stinner in :gh:`120743`.)
 
+* :mod:`pdb`:
+  The undocumented ``pdb.Pdb.curframe_locals`` is deprecated because with
+  PEP 667 we don't need to cache the locals anymore. Derived debuggers
+  should access ``pdb.Pdb.curframe.f_locals`` directly.
```

Comment:
```suggestion
  The undocumented ``pdb.Pdb.curframe_locals`` attribute is now a deprecated read-only property
  accessing ``pdb.Pdb.curframe.f_locals``. The low overhead dynamic frame locals access added in
  Python 3.13 by PEP 667 means the frame locals cache reference previously stored in this attribute
  is no longer needed. Derived debuggers should access ``pdb.Pdb.curframe.f_locals`` directly in
  Python 3.13 and later versions.
```

Reply:
Is this correct? Line 617 does not read like a correct sentence.